addons/editor-compact/userstyle.css

```diff
@@ -0,0 +1,422 @@
+/* Full page */
```

Comment:
bug: when hide-flyout is enabled, the scrollbars shows when the flyout is hidden:
![image](https://user-images.githubusercontent.com/72760579/195226384-fd30e119-4f3f-4c82-8863-94417c82f30e.png)

Reply:
Can't repro.